src/lib.rs

```diff
@@ -273,7 +275,7 @@ impl Server {
     }
 
     /// Returns the address the server is listening to.
-    #[experimental]
+    #[unstable]
```

Comment:
Note that both `#[experimental]` and `#[unstable]` have been removed from the language.
All these lines will give you an error.